### PR TITLE
macOS vs Mac

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/tutorials/first-mvc-app-xplat/start-mvc.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Introduction to ASP.NET Core MVC on Mac or Linux | Microsoft Docs
+title: Introduction to ASP.NET Core MVC on macOS, Linux or Windows | Microsoft Docs
 author: rick-anderson
-description: Getting started with ASP.NET Core MVC and Visual Studio Code on Mac and Linux
+description: Getting started with ASP.NET Core MVC and Visual Studio Code on macOS, Linux or Windows
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-mvc-app-xplat/start-mvc.md]: comma after Linux, add "using Visual Studio Code."

File: aspnetcore/tutorials/first-mvc-app-xplat/start-mvc.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ ms.technology: aspnet
 ms.prod: asp.net-core
 uid: tutorials/first-mvc-app-xplat/start-mvc
 ---
-# Getting started with ASP.NET Core MVC on Mac and Linux
+# Getting started with ASP.NET Core MVC on macOS, Linux or Windows
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-mvc-app-xplat/start-mvc.md]: Same comment for h1 as for metadata title.

File: aspnetcore/tutorials/index.md [hunk 3]
```diff
@@ -16,18 +16,18 @@ ms.prod: asp.net-core
 The following step-by-step guides for developing ASP.NET Core applications are available:
 
 ## Building web applications
-* [Introduction to ASP.NET Core MVC on Mac or Linux ](first-mvc-app-xplat/index.md)
+* [Introduction to ASP.NET Core MVC on macOS, Linux or Windows](first-mvc-app-xplat/index.md)
```
Comment (tdykstra) [inline aspnetcore/tutorials/index.md]: Same comment for link text here as for metadata title in the article itself.

File: aspnetcore/tutorials/index.md [hunk 4]
```diff
@@ -16,18 +16,18 @@ ms.prod: asp.net-core
...
 * [Introduction to ASP.NET Core MVC with Visual Studio](first-mvc-app/index.md)
 * [Getting started with ASP.NET Core and Entity Framework Core using Visual Studio](../data/ef-mvc/index.md)
-* [Your first ASP.NET Core application on a Mac or Linux using Visual Studio Code](your-first-mac-aspnet.md)
+* [Your first ASP.NET Core application on a macOS or Linux using Visual Studio Code](your-first-mac-aspnet.md)
```
Comment (tdykstra) [inline aspnetcore/tutorials/index.md]: on ~~a~~ macOS

This is now very similar to xplat MVC - can we provide guidance on which to choose somehow?  Should they do this one first, so we would put it first in the list?  Or just if they're short on time?  What does it give that they can't get in xplat MVC? Should we retire this one?

File: aspnetcore/tutorials/first-mvc-app-xplat/start-mvc.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Introduction to ASP.NET Core MVC on Mac or Linux | Microsoft Docs
+title: Introduction to ASP.NET Core MVC on macOS, Linux or Windows | Microsoft Docs
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app-xplat/start-mvc.md]: I agree - but I'd like to get the SEO juice of having macOS & Linux in the title. But that's not technically accurate to leave out Windows. For now I'll leave out Windows as Visual Studio is the preferred approach to development on Windows. 

**Introduction to ASP.NET Core MVC  using VS Code** doesn't let folks know this works on macOS and Linux
cc /@danroth27

**Pull request closed without merging**